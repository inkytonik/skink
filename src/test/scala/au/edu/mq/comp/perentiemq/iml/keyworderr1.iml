// Keyword as identifier
def if(a): (b) = {
  a = b;
}
