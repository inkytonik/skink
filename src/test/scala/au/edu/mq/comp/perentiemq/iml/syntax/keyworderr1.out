src/test/scala/au/edu/mq/comp/perentiemq/iml/syntax/keyworderr1.iml:2:5: identifier expected
def if(a): (b) = {
    ^
